tests/integration/gateway_clients/test_long_flow_keepalive.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+import pytest
+from docarray import DocumentArray
+
+from jina import Executor, Flow, requests
+
+
+@pytest.fixture()
+def slow_executor() -> Executor:
+    class MySlowExec(Executor):
+        @requests
+        def slow(self, docs, **kwargs):
+            time.sleep(300)
```

Comment:
can we reduce this ? just to ease our CI

Reply:
good point, done